_haikus/omnitrix_10.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Omnitrix
+author: Ben
+---
+
+
+Ben10 is super cool<br>
```

Comment:
Not sure about this one, is `Ben10` one syllable? How is it pronounced?

Reply:
it is also 1, totally 5
https://syllablecounter.net/count
https://www.howmanysyllables.com/syllable_counter/

Reply:
How do you pronounce it?